service_container/service_subscribers_locators.rst

```diff
@@ -140,6 +140,11 @@ count and iterate over the services of the locator::
     The :class:`Symfony\\Contracts\\Service\\ServiceCollectionInterface` was
     introduced in Symfony 7.1.
 
+Without autoconfiguration
+~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
not sure we need a headline here, WDYT of
```rst
.. note::

    If you are not using autoconfiguration you need to require either ``Psr\Container\ContainerInterface`` or ``Symfony\Contracts\Service\ServiceProviderInterface`` as a ``__construct`` argument or as a method call like ``setContainer(Psr\Container\ContainerInterface $container)``.
```

Reply:
Yes I think you are right. I started like this and changed for no good reason. will update